components/script/script_thread.rs

```diff
@@ -1267,6 +1267,12 @@ impl ScriptThread {
             #[cfg(feature = "webgpu")]
             document.update_rendering_of_webgpu_canvases();
 
+            if document
```

Comment:
It would be good to document what this step represents, for example, could this be considered part of the "update the rendering or user interface of doc" step that is documented below? And if so, it would be good to add a note explaining why it has to be done before reflow(or otherwise explaining why the step 22 has to be broken-up in multiple code steps following a specific order.